### ISSUE TITLE
presto-jdbc can support delete operation ?

### ISSUE THREAD

**Issue by hqisog** [prestodb/presto#14737]
i  find  the presto connector of mysql couldn't support delete operation.  and i find presto-jdbc  have not provided interface for delete operation . how can i achieve it ?

**Comment by mbasmanova** [comment]
This is a known limitation: https://prestodb.io/docs/current/connector/mysql.html

@jainxrohit @highker Anything to add here?

**Comment by highker** [comment]
> This is a known limitation: https://prestodb.io/docs/current/connector/mysql.html

That is correct. Actually, that shouldn't be hard as jdbc client construct SQLs to pass down to the storage engine.

Anyone feel free to grab the tasks.

**Comment by agrawalreetika** [comment]
Hi, I would like to work on this task. Could someone please help me understand with the requirement of this task? As a part of this are we looking to enable DELETE only for mysql connector?

**Comment by hqisog** [comment]
@agrawalreetika  hey， are you start the task ?  Can I come in on the issue ?

**Comment by agrawalreetika** [comment]
@hqisog I haven't done much on this. If you want, you can pick it up.
